index.bs

```diff
@@ -2117,7 +2117,49 @@ If one were to write the <code>ReadableByteStreamController</code> class in some
 
 <h4 id="rbs-controller-internal-slots">Internal Slots</h4>
 
-TBA
+Instances of <code>ReadableByteStreamController</code> are created with the internal slots described in the following table:
+
+<table>
+  <thead>
+    <tr>
+      <th>Internal Slot</th>
+      <th>Description (<em>non-normative</em>)</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>\[[controlledReadableByteStream]]
+    <td>The <a href="#rbs-class"><code>ReadableByteStream</code></a> instance controlled
+  </tr>
+  <tr>
+    <td>\[[underlyingByteSource]]
+    <td>An object representation of the stream's underlying byte source
+  </tr>
+  <tr>
+    <td>\[[pullAgain]]
+    <td>A boolean flag indicating that pulling from the underlying byte source should be made once again
+  </tr>
+  <tr>
+    <td>\[[pulling]]
+    <td>A boolean flag indicating that the underlying byte source is currently being pulled
+  </tr>
+  <tr>
+    <td>\[[pendingPullIntos]]
+    <td>A List representing the stream's internal queue of pending pullInto operations
+  </tr>
+  <tr>
+    <td>\[[queue]]
+    <td>A List representing the stream's internal queue of chunks
```

Comment:
Is it fine to wrap all `chunk` occurrences in a paragraph(s), or only the first one like dictionary?